storybookjs/storybook
# Addons: Disable option for addon tab

Issue: #6849

## What I did
The issue above ask for a feature to disable a11y panel for a single story. But the idea of this PR is to provide a solution at a common level instead of at addon level.

The principle : 
* addons register their tab panel, passing the parameter key
* user can then disable a panel for a group of stories or a specific story
```javascript
storiesOf('My Component', module)
  // disable the a11y tab for all "My Component" stories
  .addParameters({
    a11y: { disabled: true },
  })
  .add('story 1', story1)
  .add('story 2', story2);
```

```
storiesOf('My Component', module)
  // disable the a11y tab for only "story 1"
  .add('story 1', story1, {  a11y: { disabled: true } })
  .add('story 2', story2);
```

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps? Yes, a `Addon panel` example has been added
- Does this need an update to the documentation? Yes, it's a common addon behavior, but addons need to pass the param key now. Documentation has been added.

If your answer is yes to any of these, please make sure to include it in your PR.
